modules/candidate_parameters/css/candidate_parameters.css

```diff
@@ -4,4 +4,9 @@
 
 .nav-tabs {
     margin-left: 0px;
+}
+
```

Comment:
Table history seems to be more readable without this class IMHO.
Also `.table-history` is not much longer than `.table-hist`